pytorch_lightning/trainer/optimizers.py

```diff
@@ -141,26 +141,6 @@ def configure_schedulers(self, schedulers: list, monitor: Optional[str] = None):
                 raise ValueError(f'The provided lr scheduler "{scheduler}" is invalid')
         return lr_schedulers
 
-    def reinit_scheduler_properties(self, optimizers: list, schedulers: list):
```

Comment:
Where did that go ?

Reply:
this is now part of the apex amp plugin since this is the only part in the code this was used.

Reply:
unused, it is also gone on master, will update the branch once master changes are merged to 1.2ddev